src/python/WMComponent/WorkflowUpdater/WorkflowUpdaterPoller.py

```diff
@@ -64,7 +288,8 @@ def __init__(self, config):
         self.rucio = Rucio(acct=self.rucioAcct,
                            hostUrl=self.rucioUrl,
                            authUrl=self.rucioAuthUrl)
-                           # configDict={'logger': self.logger})
+        # define where to look for sandbox tar balls
+        self.sandboxDir = getattr(self.config.WorkflowUpdater, "sandboxDir")
```

Comment:
As previously mentioned, we do not need this component attribute. Workflow sandbox path comes straight from one of the database calls (for the spec path). The least component attributes the better.

Reply:
ok, removed